src/card.css

```diff
@@ -0,0 +1,88 @@
+.card--content {
+  width: 200px;
```

Comment:
put `width` in `.card` block

Reply:
https://github.com/mate-academy/layout_product-cards/pull/2032#discussion_r1000818127
7. [BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.
So what should I do and whom to listen?